### ISSUE TITLE
Navigate to File Manager directory based on page route

### ISSUE THREAD

**Issue by iGalaxyYT** [Minehut/Meta#45]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website (File Manager)

**Describe the feature**

When the file manager page loads, go to a directory based on the current route.
Example:
https://minehut.com/dashboard/files/ -> Root directory
https://minehut.com/dashboard/files/plugins/Skript/scripts -> (root)/plugins/Skript/scripts

**How the feature is useful**

This would allow users to easily "bookmark" certain directories on their browser as well as navigate to a directory that they remember without having to use the visual interface.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by drtshock** [comment]
Yeah I think we could do that. We would need to have it be absolute path based on server though. So would likely be `/dashboard/<serverid>/restofpath`
